plugin.xml

```diff
@@ -37,10 +37,14 @@
     <!-- android -->
     <platform name="android">
         <config-file target="res/xml/config.xml" parent="/*">
-            <feature name="Device" >
+            <feature name="Device">
                 <param name="android-package" value="org.apache.cordova.device.Device"/>
             </feature>
         </config-file>
+        <config-file target="AndroidManifest.xml" parent="/*">
+            <uses-permission android:name="android.permission.READ_PHONE_STATE"/>
+            <uses-permission android:name="android.permission.READ_PRIVILEGED_PHONE_STATE"/>
```

Comment:
Cordova cannot define privileged permissions because that would cause apps to be rejected from the Google Play store.